### PR TITLE
Cortx 31882 putobject tagging happy path

### PR DESCRIPTION
**Pull request by akshaym99**
# Problem Statement
- Automate testcases for F28B: PutObjectTagging - Happy path

# Design
-  TD doc: https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/983465985/F-28B+S3+Object+Tagging+API 

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by akshaym99** [comment]
@apoorva-rao , @DhobaleVishal , @VikasKumarC Please review.

**Review by apoorva-rao — APPROVED** [review]

File: tests/s3/test_object_tagging.py [hunk 1]
```diff
@@ -759,7 +759,7 @@ def test_valid_specialchars_2473(self):
             self.object_name,
             self.file_path,
             mb_count=S3_OBJ_TST["s3_object"]["mb_count"],
-            key=S3_OBJ_TST["s3_object"]["key"],
+            key=S3_OBJ_TST["test_9429"]["key"],
```
Comment (DhobaleVishal) [inline tests/s3/test_object_tagging.py:762]: Is this related to changes from yaml?

File: tests/s3/test_object_tagging.py [hunk 2]
```diff
@@ -984,10 +984,11 @@ def test_duplicate_name_object_tag_support_2477(self):
...
-    def test_max_object_max_tags_2478(self):
-        """Verification of max. no. of Objects user can upload with max no. of tags per Object."""
-        self.log.info(
-            "Verification of max. no. of Objects user can upload with max no. of tags per Object")
+    def test_multiple_object_max_tags_2478(self):
+        """Verification of multiple no. of Objects user
+        can upload with max no. of tags per Object."""
+        self.log.info("Verification of multiple no. of Objects "
+                      "user can upload with max no. of tags per Object")
         self.log.info(
```
Comment (DhobaleVishal) [inline tests/s3/test_object_tagging.py:992]: We follow line length limit up to 100 characters, can you please follow same to reduce number of lines from this test module?

**Review by DhobaleVishal — CHANGES_REQUESTED** [review]

File: tests/s3/test_object_tagging.py (hunk 1 above)
Comment (akshaym99) [reply tests/s3/test_object_tagging.py:762]: Yes, taking tag key as valid special characters.

File: tests/s3/test_object_tagging.py (hunk 2 above)
Comment (akshaym99) [reply tests/s3/test_object_tagging.py:992]: Addressed & updated.

**Review by DhobaleVishal — APPROVED** [review]
LGTM

**Review by VikasKumarC — APPROVED** [review]

**Review by sonalk0209 — APPROVED** [review]
Approving for merge!!!
